waffle_hub/dataset/dataset.py

```diff
@@ -1114,6 +1276,12 @@ def check_trainable(self):
         Raises:
             ValueError: if dataset has not enough annotations.
         """
+        if self.task == TaskType.TEXT_RECOGNITION:
+            if len(self.images) < Dataset.MINIMUM_TRAINABLE_IMAGE_NUM_PER_CATEGORY:
+                raise ValueError(
+                    f"number of image is bigger than MINIMUM_TRAINABLE_IMAGE_NUM_PER_CATEGORY({Dataset.MINIMUM_TRAINABLE_IMAGE_NUM_PER_CATEGORY})"
+                )
+            return
```

Comment:
`is bigger than` -> `should be larger than`

By the way, why did you seperate this trainable check with other tasks?
I think the number of some categories can be zero.
Is it okay in ocr task?

Reply:
Then, we should implement additional function to cover it, or enhace to `categroy_to_image` logic.

For now, I can not accept this code because of the risk I said.